repo/packages/C/calico/0/marathon.json.mustache

```diff
@@ -0,0 +1,55 @@
+{
+  "id": "calico",
+  "container": {
+    "type": "DOCKER",
+    "network": "HOST",
+    "docker": {
+      "image": "{{resource.assets.container.docker.calico-dcos}}",
+      "network": "HOST",
+      "forcePullImage": true
+    }
+  },
+  "args": [],
+  "cpus": {{calico.cpu-limit-framework}},
+  "mem": {{calico.mem-limit-framework}},
+  "instances": 1,
+  "env": {
+    "CALICO_CALICOCTL_URL": "{{calico.calicoctl-url}}",
```

Comment:
Ah I gotcha now - I assume then we also need to move the calico0node and calico-libnetwork images to the resources as well - since these are container images